server/src/test/java/org/elasticsearch/index/mapper/IpFieldMapperTests.java

```diff
@@ -28,61 +28,28 @@
 import org.apache.lucene.util.BytesRef;
```

Comment:
This also converts IpFieldMapperTests to MapperTestCase because it just makes testing so much easier...